src/components/molecules/image-crop-dialog/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import 'react-image-crop/dist/ReactCrop.css'
+
+import ImageCropper from '_molecules/image-cropper'
+import Dialog from '_molecules/dialog'
+import ErrorMessageList from '_molecules/error-message-list'
+
+const ImageCropDialog = ({ invalidProperties, selectedFile, crop, ...props }) => (
+  <Dialog {...props}>
```

Comment:
pass props forward using `...props` is not a best practice, try to name all props instead.